src/adapter/input/blog_router.py

```
@@ -0,0 +1,34 @@
+def get_post_application_service():
+    db = SessionLocal()
+    try:
+        adapter = VisitRepositoryImpl(db)
+        usecase = VisitUseCase(adapter)
+        service = VisitApplicationService(usecase)
+        yield service
+    finally:
+        db.close()
+
+@router.get('/blog')
+def get_blog_main(request:Request, serivce:VisitApplicationService = Depends(get_post_application_service)):
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)